stories/primer/dropdown/menu_component_stories.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "primer/dropdown/menu_component"
```

Comment:
for some reason, the story can't find `Primer::Dropdown::MenuComponent` without this require